xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/Main/SpaceIndex.xml

```diff
@@ -37,12 +37,20 @@
   <syntaxId>xwiki/2.0</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-#if("$!request.space" != "") #set($space=$request.space) #else #set($space = $doc.space) #end
-$services.localization.render('platform.index.spaceIndexDescription', [$space])
+#if("$!request.space" != "")
+  #set($space=$request.space)
+#else
+  #set($space = $doc.space)
+#end
 ## Since we display the space index, we do not display page titles and locations, but the old columns name and space 
 ## (old behavior consistent with this old macro)...
 
-{{documents space="$space.replaceAll('["~]', '~$0')" id="spaceindex" columns="doc.name, doc.space, doc.date, doc.author"/}}
+{{documents
+  space="$space.replaceAll('["~]', '~$0')"
+  id="spaceindex"
+  columns="doc.name,doc.space,doc.date,doc.author"
+  description="$services.localization.render('platform.index.spaceIndexDescription', [$space])"
```

Comment:
This is an XWiki syntax injection vulnerability, `$space` needs to be escaped here - at least in the way it is escaped above.

Reply:
Done in https://github.com/xwiki/xwiki-platform/pull/2775/commits/c821d6f0da1de98c3818980ea310bed501a7a66d